File: examples/sorting-algorithms/case-already-sorted.ts
const data = Array.from({ length: 1000 }, (_, i) => i);

export function createCase() {
  return {
    case: {
      caseId: "already-sorted",
      caseClass: "sorted",
      name: "Already Sorted Array",
      version: "1.0.0",
      inputs: { data },
    },
    getInput: async () => ({ data }),
    getInputs: () => ({ data }),
  };
}

File: examples/sorting-algorithms/case_already_sorted.py
"""Already Sorted Array case — [0..999]."""

_data = list(range(1000))


def create_case():
    return {
        "case": {
            "caseId": "already-sorted",
            "caseClass": "sorted",
            "name": "Already Sorted Array",
            "version": "1.0.0",
            "inputs": {"data": _data},
        },
        "getInput": lambda: {"data": _data},
        "getInputs": lambda: {"data": _data},
    }
